LICENSE

```diff
@@ -79,6 +80,7 @@ The Chapel implementation is composed of two categories of code:
      re2              only used when CHPL_REGEXP is 're2'
      utf8-decoder     bundled into the Chapel runtime to decode UTF-8 strings
      libunwind        only used when CHPL_UNWIND is 'libunwind'
+     whereami         used when starting a program with a launcher
```

Comment:
@gbtitus: Reviewing this makes me realize belatedly:  We should have probably added an entry to this second "when is it used?" table for libfabric?

Reply:
Oops, yeah, I'll fix that.  Thanks @bradcray!